.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Integration Tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          cache: "pip"
+          cache-dependency-path: "requirements*.txt"
+
+      - name: Install python packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade --upgrade-strategy eager -e .[dev]
+          
+      - name: Build orion-ui
+        run: |
+          prefect dev build-ui
+          
+      - name: Start orion
+        run: |
+          prefect orion start &
+
+      - name: Set up node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache-dependency-path: '**/package-lock.json'
+        
+      - name: Cache Playwright Browsers
+        id: cache-playwright-browsers
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-browsers
```

Comment:
Nice! Looks like this is working well. You may want to retrieve the playwright version in a prior step for the `key`.

Reply:
Yes I gotta figure that piece out. Although I did find this in the playwright repository which makes me wonder if this is sufficient. https://github.com/microsoft/playwright-github-action/actions/runs/91812666/workflow

Reply:
The [npm example](https://github.com/actions/cache/blob/main/examples.md#java---maven) shows how to use the result of a previous command downstream. Interesting that the docs say you should use the playright version. FWIW the cache in the workflow you linked is a couple years old.

Reply:
Think I've got that working now